utils.sh

```diff
@@ -494,4 +494,23 @@ function auth_template_and_removetmp(){
     removetmp
 }
 
+function configure_samples_operator() {
+    REGISTRY_CONFIG="registry-config"  
+    oc create configmap ${REGISTRY_CONFIG} \
+      --from-file=${LOCAL_REGISTRY_DNS_NAME}..${LOCAL_REGISTRY_PORT}=${REGISTRY_DIR}/certs/${REGISTRY_CRT} -n openshift-config
+    oc patch image.config.openshift.io/cluster --patch "{\"spec\":{\"additionalTrustedCA\":{\"name\":\"${REGISTRY_CONFIG}\"}}}" --type=merge
```

Comment:
Can you move the above two lines in a separate function in `utils.sh` near [`setup_local_registry()`](https://github.com/openshift-metal3/dev-scripts/blob/831962cce8e140f95572041ddf7c87352dc3d864/utils.sh#L328)?